scripts/aleth.py

```diff
@@ -22,7 +22,7 @@
 
 try:
     if wrapper_args.rpc:
-        run_daemon()
+        run_daemon(wrapper_args.rpc)
```

Comment:
Just realized that this will not work if `aleth.py` will be called with `--db-path` or `--ipcpath`.
Here for this container it will be fine of course

Reply:
The `--db-path` will work, but it will be a path inside the docker.

I don't want to put a lot of effort to make it perfectly robust now, however the idea of having aleth extendable with Python scripting is quite nice (but probably with different architecture).
Anyway, I will try to quickly intercept `--ipcpath` and `--no-ipc` flags.

Reply:
I meant that `--dp-path` also changes the IPC socket path

Reply:
Ok, `--db-path` also handled.